hocuspocus-extension-elasticsearch/src/__test__/elasticsearch.test.ts

```diff
@@ -0,0 +1,86 @@
+import { newHocuspocus, newHocuspocusProvider, delayForMs } from 'test/utils';
+import { Elasticsearch } from '../elasticsearch';
+import elasticsearch from '@elastic/elasticsearch';
+
+describe('elasticsearch extension', () => {
+  let server;
+  const elasticsearchOpts: elasticsearch.ClientOptions = {
+    node: process.env.ELASTICSEARCH_URL,
+  };
+  beforeAll(async () => {
+    server = await newHocuspocus({
+      yDocOptions: { gc: false, gcFilter: () => true },
+      port: process.env.PORT ? Number(process.env.PORT) : 1234,
+
+      extensions: [
+        new Elasticsearch({
+          elasticsearchOpts,
+        }),
+      ],
+    });
+  });
+  afterAll(async () => {
+    await server.destroy();
+    await delayForMs(1000);
+  });
+
+  it('return default ydoc when fetched documentName does not exist', (resolve) => {
+    const provider = newHocuspocusProvider(server, {
+      onSynced() {
+        expect(provider.document.share.size).toBe(0);
```

Comment:
Question: what is `document.share.size` here?

Reply:
[`document.share`](https://github.com/yjs/yjs/blob/fe36ffd122a6f2384293098afd52d2c0025fce2a/src/utils/Doc.js#L53C6-L53C6) is the base `Map` of [shared types](https://github.com/yjs/yjs#shared-types).